### PR TITLE
Maya: Add is_static_image_plane and is_in_all_views option in imagePlaneLoader

### PR DESCRIPTION
**Pull request by ClementHector**
I need to import images published of design in maya on the front and side view to serve as blueprint for the modeling. We add the optional possibility of having fixed and visible plane images in all windows.

### CODE REVIEW

**Review by tokejepsen — APPROVED** [review]
Looks good to me. Cant test unfortunately.

**Review by antirotor — APPROVED** [review]
I does work perfectly, thanks.

File: openpype/hosts/maya/plugins/load/load_image_plane.py [hunk 1]
```diff
@@ -13,10 +13,14 @@ def __init__(self, cameras):
         self.setWindowFlags(self.windowFlags() | QtCore.Qt.FramelessWindowHint)
 
         self.camera = None
+        self.static_image_plane = False
+        self.show_in_all_views = False
```
Comment (tokejepsen) [inline openpype/hosts/maya/plugins/load/load_image_plane.py:17]: Does this mean its off by default in the UI?

**Review by tokejepsen — CHANGES_REQUESTED** [review]

File: openpype/hosts/maya/plugins/load/load_image_plane.py (hunk 1 above)
Comment (ClementHector) [reply openpype/hosts/maya/plugins/load/load_image_plane.py:17]: Exactly.

**Review by tokejepsen — APPROVED** [review]
Looks good to me. Cant test it unfortunately.

**Pull request merged**